chsdi/templates/htmlpopup/laerm_eisenbahn_zulaessige_immissionen_nacht.mako

```diff
@@ -0,0 +1,18 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+<%
+  lang = lang if lang in ('fr','it') else 'de'
+  lang = lang if lang != 'it' else 'fr'
+  es = '%s_es' % lang
+  pointofdetermination_t = '%s_pointofdetermination_t' % lang
+%>
+
+<% c['stable_id'] = True %>
```

Comment:
you should not use stable_id TRUE if the primary key is an db sequence. This should option should only be used if the primary key is part of the data delivery and declared as stable by data owner.

Reply:
> you should not use stable_id TRUE if the primary key is an db sequence. This should option should only be used if the primary key is part of the data delivery and declared as stable by data owner.

OK thanks for your feedback. I will adapt it then